Libraries/LibGUI/ScrollBar.cpp

```diff
@@ -277,25 +292,23 @@ void ScrollBar::mousedown_event(MouseEvent& event)
         return;
     }
 
-    if (has_scrubber()) {
```

Comment:
No functional changes here, just swapped this `if` statement for an early return above. Diff is slightly messed up.